Simplenote/Classes/SPResultsController.swift

```diff
@@ -0,0 +1,150 @@
+import Foundation
+import CoreData
+
+
+// MARK: - SPResultsController
+//
+class SPResultsController {
```

Comment:
Maybe tweak the naming to be specific to search? `SPResultsController` sounds like it might be a generic wrapper around an FRC.  `SPSearchResultsController` could work but its similar to `SPSearchResultsViewController`. Maybe that's not so bad?